starter-code/views/celebs/celeb-edit.hbs

```diff
@@ -0,0 +1,15 @@
+<h2>Edit the Celebrity: {{name}}</h2>
+<form action="/celebrities/{{_id}}/edit" method="post">
+    <label for="title-input">Name:</label>
```

Comment:
the "for" attribute of the label tag should have the same value as the id of the input it is targeting! In this case, it should be "name-input" 👍

Reply:
Thanks Filipe I miss that one 😁😖